src/components/ActivityFeed/ActivityFeedTreasury.tsx

```
@@ -250,153 +120,385 @@ const ActivityFeedSidebar = () => {
+									)
+								);
+							}
+						} catch (error) {
+							console.log(error);
+						}
+						setNextBurn({
+							isLoading: false,
+							value,
+							valueUSD
+						});
+					}
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)